specification/modsimworkbench/resource-manager/Microsoft.ModSimWorkbench/preview/2021-03-01-preview/chambers.json

```diff
@@ -0,0 +1,1042 @@
+        "enable": {
+          "type": "boolean",
+          "description": "Switch to show if License Service is activated."
+        },
+        "server": {
+          "type": "string",
+          "description": "License Service server."
+        },
+        "lmhostId": {
+          "type": "array",
+          "description": "Unique identifier of ModSim Workbench chamber license servers used by FlexNet Licensing Service.",
+          "items": {
+            "type": "string"
+          },
+          "x-ms-identifiers": [
+            "lmhostId"
+          ]
+        }
+      }
+    },
+    "Synopsys": {
+      "type": "object",
+      "description": "Synopsys Common Licensing",
+      "properties": {
+        "enable": {
+          "type": "boolean",
+          "description": "Switch to show if License Service is activated."
+        },
+        "server": {
+          "type": "string",
+          "description": "License Service server."
+        },
+        "vmUuid": {
+          "type": "array",
+          "description": "Virtual machine universal unique identifier of ModSim Workbench chamber license servers used by FlexNet Licensing Service.",
+          "items": {
+            "type": "string"
+          },
+          "x-ms-identifiers": [
+            "vmUuid"
+          ]
+        }
+      }
+    },
+    "ChamberPropertiesFormat": {
+      "type": "object",
+      "properties": {
+        "licenseServices": {
+          "x-ms-client-flatten": true,
+          "readOnly": true,
+          "description": "Gets or sets the license services for each EDA vendor.",
+          "type": "object",
+          "properties": {
+            "cadence": {
+              "x-ms-client-flatten": true,
+              "$ref": "#/definitions/Cadence",
+              "description": "Gets or sets the cadence properties."
+            },
+            "ansys": {
+              "x-ms-client-flatten": true,
+              "$ref": "#/definitions/Ansys",
+              "description": "Gets or sets the ansys properties."
+            },
+            "siemens": {
+              "x-ms-client-flatten": true,
+              "$ref": "#/definitions/Siemens",
+              "description": "Gets or sets the siemens properties."
+            },
+            "synopsys": {
+              "x-ms-client-flatten": true,
+              "$ref": "#/definitions/Synopsys",
+              "description": "Gets or sets the synopsys properties."
+            }
+          }
+        },
+        "provisioningState": {
+          "$ref": "./modsimworkbenches.json#/definitions/ProvisioningState",
+          "description": "Gets the status of the Chamber at the time the operation was called"
+        }
+      },
+      "description": "Chamber properties."
+    },
+    "GetUploadUri": {
+      "description": "Datasync file upload SAS token",
+      "type": "object",
+      "properties": {
+        "result": {
+          "type": "string",
+          "description": "Datasync file upload SAS token"
```

Comment:
>description [](http://example.com/codeflow?start=11&length=11)

SAS tokens are secrets, to better mark them as x-ms-secret everywhere

Reply:
sasUriId value is not SAS URI, it's key vault secret ID for the SAS URI: not secretss